sample.mvn.parent/pom.xml

```diff
@@ -83,10 +147,10 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.8.0</version>
+				<version>${maven.plugins-maven-compiler-plugin-version}</version>
 				<configuration>
-					<source>11</source>
-					<target>11</target>
+					<source>${maven.plugins-maven-compiler-plugin.source}</source>
```

Comment:
We could define the following 2 properties and leave out the configuration section completely
* <maven.compiler.source>11</maven.compiler.source>
* <maven.compiler.target>11</maven.compiler.target>

Reply:
done